Update documentation for metoffice to support migration to DataHub API (#35954)
home-assistant/home-assistant.github.io: source/_integrations/metoffice.markdown

ha_integration_type: integration
---

> The `metoffice` weather platform uses the Met Office's [DataHub API](https://www.metoffice.gov.uk/services/data/met-office-weather-datahub) for weather data.

Reviewer: ```suggestion
The **Met Office** weather {% term integration %} uses the Met Office's [DataHub API](https://www.metoffice.gov.uk/services/data/met-office-weather-datahub) for weather data.
```

> ## Getting started

Reviewer: ```suggestion
## Prerequisites
```

1. Register for a [Met Office DataHub](https://datahub.metoffice.gov.uk/) account.

> 2. After registration, [subscribe](https://datahub.metoffice.gov.uk/profile/subscriptions) to [Site Specific Global Spot](https://datahub.metoffice.gov.uk/pricing/site-specific) dataset to obtain your API key. Free tier subscription gives 360 calls per day which is enough for using this integration

Reviewer: ```suggestion
2. After registration, [subscribe](https://datahub.metoffice.gov.uk/profile/subscriptions) to [Site Specific Global Spot](https://datahub.metoffice.gov.uk/pricing/site-specific) dataset to obtain your API key. Free tier subscription gives 360 calls per day, which is enough for using this integration
```

## Getting started
1. Register for a [Met Office DataHub](https://datahub.metoffice.gov.uk/) account. 
2. After registration, [subscribe](https://datahub.metoffice.gov.uk/profile/subscriptions) to [Site Specific Global Spot](https://datahub.metoffice.gov.uk/pricing/site-specific) dataset to obtain your API key. Free tier subscription gives 360 calls per day which is enough for using this integration

{% include integrations/config_flow.md %}

## Entities

This integration creates a number of weather entities for each entry created in the configuration by location: one weather entity with a summary and a forecast (twice-daily and hourly), and twelve sensor entities for individual reporting on each of the individual measurements. Note that only some of the sensor entities flagged below are enabled by default, so your system isn't overrun on initial configuration.

The available sensor entities:

- "feels like" temperature
- humidity
- pressure
- probability of precipitation
- station name
- temperature
- UV index
- visibility
- weather
- wind direction
- wind gust
- wind speed

Only probability of precipitation, pressure, temperature, weather and wind speed are enabled by default.

> Details about the API are available in the [DataHub API documentation](https://datahub.metoffice.gov.uk/docs/f/category/site-specific/overview). The [datapoint-python](https://github.com/EJEP/datapoint-python) library is used to retrieve data.

Author: This suggestion is wrong. `datapoint-python` version gets updated to 0.12.1 which doesn't have compatibility issues and uses DataHub API.
